ns1/resource_record_test.go

```diff
@@ -828,7 +845,7 @@ resource "ns1_monitoringjob" "test" {
     "ams"
   ]
   job_type = "http"
-  frequency = 30
+  frequency = 60
```

Comment:
what's this change for?

Reply:
In the tests, there was a error where my account couldn't create monitor jobs with less than 60 frequency